ocamlformat.opam

```diff
@@ -39,5 +26,17 @@ depends: [
   "uuseg" {>= "10.0.0"}
   "uutf" {>= "1.0.1"}
 ]
-synopsis: "Auto-formatter for OCaml code"
-description: "OCamlFormat is a tool to automatically format OCaml code in a uniform style."
+dev-repo: "git+https://github.com/ocaml-ppx/ocamlformat.git"
+build: [
+  ["ocaml" "tools/gen_version.mlt" "lib/Version.ml" version] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
Is it obvious that these have the same behavior as the version before this PR?

Reply:
Ok, I hadn't appreciated that `{with-test}` could be placed on individual arguments instead of whole commands.

Ok, so `@install` only builds the files to be installed, but does not actually copy them into some other directories, etc. Correct?

Reply:
I will ask on discord for confirmation.

Reply:
Yes they are doing almost the same thing. As written the command build all installable libraries executables (Even if not needed for the tests) and runs the tests. You can add different filters on each string in the list In the opam files, it’s quite convenient at times